addok/batch.py

```diff
@@ -15,11 +17,24 @@ def run(args):
         process_stdin(sys.stdin)
 
 
+def flush(args):
```

Comment:
Maybe `flushdb` then? Which is what we do Redis side?
And we may also rename the DS method.